utility/BUILD.bazel

```diff
@@ -17,6 +17,22 @@ MIGRATED_PROD_FILES = glob([
     "src/main/java/org/oppia/android/util/system/*.kt",
 ])
 
+filegroup(
+    name = "manifest",
+    srcs = [
+        "src/main/AndroidManifest.xml",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+android_library(
+    name = "strings",
```

Comment:
Could we export all resources instead of just strings? That would use a ``glob`` below for the ``resource_files``, and this should be called "resources".

Please also remove the resource files & manifest reference from "utility" below. If it still needs resources then it should add a dependency on this, instead.

Reply:
done